frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.unit.spec.tsx

```diff
@@ -163,7 +163,8 @@ function setup(additionalProps?: Partial<ExpressionWidgetProps>) {
     onChangeExpression: jest.fn(),
   };
 
-  const props = {
+  // we test only legacy queries for now
+  const props: Omit<ExpressionWidgetProps, "query" | "stageIndex"> = {
```

Comment:
FYI I added tests for MLv2 query in my PR: https://github.com/metabase/metabase/pull/36031/files#diff-6996a8ef047a33b549cc21b76fea281c75d191834be45cf91d9e2b31ce3b2cb5R203